ios/RNSScreen.mm

```diff
@@ -295,6 +295,14 @@ - (void)addSubview:(UIView *)view
   }
 }
 
+- (void)postNotificationWithEvent:(NSObject<RCTEvent> *)event
```

Comment:
Also, this method could be placed in Fabric specific section & compiled only on Fabric.

Reply:
Done